docs/starters.yml

```
@@ -1634,3 +1634,14 @@
     - GSAP ready
     - Nice devtools
     - Decorators
+- url: https://gatsby-contentstack-starter.netlify.com/
+  repo: https://github.com/contentstack/gatsby-starter-contentstack
+  description: A gatsby starter using contentstack source plugin powered by Headless CMS Contentstack.
```

I think that's a bit repetitive, isn't it? Why not write:

```suggestion
  description: A Gatsby starter powered by Headless CMS Contentstack.
```